src/main/java/org/baldogru/imagegallery/exception/FailValidationException.java

```diff
@@ -0,0 +1,7 @@
+package org.baldogru.imagegallery.exception;
+
+public class FailValidationException extends RuntimeException {
+
+    public FailValidationException() {
```

Comment:
Jak domyślny konstruktor ma mieć modyfikator public to nie musisz go tworzyć na siłę :)